### PR TITLE
Add image list/detail endpoints

### PR DESCRIPTION
**Pull request by danlamanna**

### CODE REVIEW

**Comment by danlamanna** [comment]
@brianhelba this should do lookups based on isic id, and it rounds ages appropriately.

File: isic/ingest/tests/factories.py [hunk 1]
```diff
@@ -83,3 +83,5 @@ class Meta:
     phi_check = factory.Faker('boolean')
     duplicate_check = factory.Faker('boolean')
     lesion_check = factory.Faker('boolean')
+
+    metadata = factory.Dict({})
```
Comment (danlamanna) [inline isic/ingest/tests/factories.py:87]: @brianhelba This is causing every the tests to fail, but works for `test_core_api_image_ages_are_always_rounded` where I set `accession__metadata__age`. Have you done this any other way?

Comment (danlamanna) [reply isic/ingest/tests/factories.py]: fixed.

**Review by brianhelba — CHANGES_REQUESTED** [review]

**Comment by brianhelba** [comment]
I pushed one commit adding some metadata content to `AccessionFactory`.

**Comment by danlamanna** [comment]
> I pushed one commit adding some metadata content to AccessionFactory.

@brianhelba Not sure why your commit broke, it also breaks other test(s?) that assume metadata is blank. Can you take a look, or back it out of this PR which is otherwise working?

**Review by brianhelba — APPROVED** [review]

**Review by brianhelba — APPROVED** [review]

**Pull request merged**